docs/deploy/index.md

```diff
@@ -102,70 +103,78 @@ This example creates an ELB with just two listeners, one in port 80 and another
 ```
 
-### GCE - GKE
+#### GCE - GKE
 
 ```console
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/provider/cloud-generic.yaml
 ```
 
 **Important Note:** proxy protocol is not supported in GCE/GKE
 
-### Azure
+#### Azure
 
 
 ```console
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/provider/cloud-generic.yaml
 ```
 
 
-### Baremetal
+#### Baremetal
 
 Using [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport):
 
 ```console
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/provider/baremetal/service-nodeport.yaml
 ```
 
-## Using Helm
+### Verify installation
 
-NGINX Ingress controller can be installed via [Helm](https://helm.sh/) using the chart [stable/nginx](https://github.com/kubernetes/charts/tree/master/stable/nginx-ingress) from the official charts repository. 
-To install the chart with the release name `my-nginx`:
+To check if the ingress controller pods have started, run the following command:
 
 ```console
-helm install stable/nginx-ingress --name my-nginx
+kubectl get pods --all-namespaces -l app=ingress-nginx --watch
 ```
 
-If the kubernetes cluster has RBAC enabled, then run:
+Once the operator pods are running, you can cancel the above command by typing `Ctrl+C`.
+Now, you are ready to create your first ingress.
+
+### Detect installed version
+
+To detect which version of the ingress controller is running, exec into the pod and run `nginx-ingress-controller version` command.
 
 ```console
-helm install stable/nginx-ingress --name my-nginx --set rbac.create=true
+POD_NAMESPACE=ingress-nginx
+POD_NAME=$(kubectl get pods -n $POD_NAMESPACE -l app=ingress-nginx -o jsonpath={.items[0].metadata.name})
+kubectl exec -it $POD_NAME -n $POD_NAMESPACE -- /nginx-ingress-controller --version
 ```
 
-## Verify installation
+## Using Helm
 
-To check if the ingress controller pods have started, run the following command:
+NGINX Ingress controller can be installed via [Helm](https://helm.sh/) using the chart [stable/nginx](https://github.com/kubernetes/charts/tree/master/stable/nginx-ingress) from the official charts repository. 
+To install the chart with the release name `my-nginx`:
 
 ```console
-kubectl get pods --all-namespaces -l app=ingress-nginx --watch
+helm install stable/nginx-ingress --name my-nginx
 ```
 
-Once the operator pods are running, you can cancel the above command by typing `Ctrl+C`.
-Now, you are ready to create your first ingress.
+If the kubernetes cluster has RBAC enabled, then run:
 
-## Detect installed version
+```console
+helm install stable/nginx-ingress --name my-nginx --set rbac.create=true
+```
 
-To detect which version of the ingress controller is running, exec into the pod and run `nginx-ingress-controller version` command.
+Detect installed version:
 
 ```console
-POD_NAMESPACE=ingress-nginx
```

Comment:
This works only if you are running the ingress controller in the default namespace.
By default, we use the `ingress-nginx` namespace

Reply:
Correct, however the helm chart uses the `default` namespace by default. The instructions for the generic deployment are the same.